### ISSUE TITLE
Thumbnails can't displayed

### ISSUE THREAD

**Issue by omnivelox** [satanasov/phpbbgallery#268]
Hi,
This is a very nice ext for phpbb. I installed and am using. Only one issue is, thumbnails are not displayed. How can i fix it? Or, can i fix it?

In the image page, thumbnail link is broken and in the source, Error 500 "internal server error"
link: http://localhost/app.php/gallery/image/4/mini

**Comment by omnivelox** [comment]
Of course, clicking on the broken thumbnail, it can display real image...

**Comment by satanasov** [comment]
Please check your rewrite options.

**Comment by Leinad4Mind** [comment]
I am not able either to replicate this issue. I'll close for now. If this is still an issue we can still talk.
